ReClass.NET/Forms/AboutForm.Designer.cs

```diff
@@ -175,6 +175,9 @@ private void InitializeComponent()
 			this.Controls.Add(this.platformLabel);
 			this.Controls.Add(this.infoLabel);
 			this.Controls.Add(this.bannerBox);
+			this.FormBorderStyle = System.Windows.Forms.FormBorderStyle.FixedDialog;
+			this.MaximizeBox = false;
```

Comment:
You don't need to disable MaximizeBox and MinimizeBox with the FixedDialog. If you remove them, I will merge the changes.

Reply:
As far as I see the MaximizeBox and MinimizeBox is still available even if FormBorderStyle is set to FixedDialog or FixedSingle. I can make short GIF if necessary.

Reply:
You are right. My bad.